src/watch_history_console.py

```diff
@@ -12,13 +12,13 @@
 from classes.whdata import WatchHistoryDataHandler as whdh
 from classes.whexcel import ExcelBuilder as excel
 
-@staticmethod
```

Comment:
in order for a method to be static, it has to be part of a class. these functions are not part of a class, so annotating them with `@staticmethod` is sort of pointless.